WatchApp Extension/Controllers/AddCarbsInterfaceController.swift

```diff
@@ -173,11 +180,13 @@ final class AddCarbsInterfaceController: WKInterfaceController, IdentifiableClas
                 try WCSession.default.sendCarbEntryMessage(entry,
                     replyHandler: { (suggestion) in
                         DispatchQueue.main.async {
+                            WKInterfaceDevice.current().play(.success)
```

Comment:
@ps2 glad to see this success haptic made it in. Would love to also see that occur in the reply handler for `sendBolusMessage` in the `BolusInterfaceController`. Any reason not to also have it there?